pkg/apis/monitoring/v1/alertmanager_types.go

```diff
@@ -273,6 +273,9 @@ type AlertmanagerGlobalConfig struct {
 
 	// HTTP client configuration.
 	HTTPConfig *HTTPConfig `json:"httpConfig,omitempty"`
+
+	// Slack Api Url
+	SlackAPIURL *string `json:"slackApiUrl,omitempty"`
```

Comment:
This needs to be a `*SecretKeySelector` type because the Slack API URL can't be stored in clear-text.

Reply:
Alright, I was a bit confused tho...

Reply:
I will change it..